test/transfer.test.ts

```diff
@@ -0,0 +1,124 @@
+import { describe, test, afterEach, before } from "mocha";
+import { spy, restore, assert, match } from "sinon";
+import yargs from "yargs/yargs";
+import { getAccounts, getDatabase } from "@tableland/local";
+import * as mod from "../src/commands/transfer.js";
+import { wait } from "../src/utils.js";
+import { helpers } from "@tableland/sdk";
+
+describe("commands/transfer", function () {
+  this.timeout("30s");
+
+  const accounts = getAccounts();
+  // account 0 is the Validator's wallet, try to avoid using that
+  const db = getDatabase(accounts[1]);
+  let tableName: string;
+  before(async function () {
+    await wait(500);
+    const { meta } = await db
+      .prepare("CREATE TABLE test_transfer (a int);")
+      .all();
+    tableName = meta.txn?.name ?? "";
+  });
+
+  afterEach(function () {
+    restore();
+  });
+
+  test("throws without privateKey", async function () {
+    const consoleError = spy(console, "error");
+    await yargs(["transfer", tableName, "0x0000000000000000000000"])
+      .command(mod)
+      .parse();
+    assert.calledWith(
+      consoleError,
+      "No registry. This may be because you did not specify a private key with which to interact with the registry."
+    );
+  });
+
+  test("throws with invalid chain", async function () {
+    const [, account] = getAccounts();
```

Comment:
This is probably my fault since I helped with the tests, but I noticed that since we already got the accounts in the 
outer scope we don't have to re-call `getAccounts`.
We could replace these calls with `const account = accounts[1];`, or something that gets the existing Wallet instead of creating 25 new Wallets.

Reply:
Good eye, fixed.